content/departments/tech-ops/tools/computer-setup.md

```diff
@@ -63,3 +68,15 @@ If you have a device that you are having replaced, or you are leaving the compan
 - Are there any issues with the laptop?
 - Would you like a box and a label shipped to you or just the label (this will be emailed)?
 - Address and phone number for the shipping
+
+## What if I have a business need for Windows or Linux
+
+- For those devices where a business need is required for a Windows environment (i.e. to run certain applications), we will create a virtual environment through the use of [Amazon Workspaces](https://aws.amazon.com/workspaces/) and show each user how they can use this on the Apple device.
+- We can create a similar Linux environment for those wishing to have this capability on their Apple devices.
+- These environments are compliant with the many certifications we are aiming to get not just SOC 2, can be better managed and more easily audited than individual devices
```

Comment:
I wouldn't say this has anything to do with the certifications per se - Maybe we can say something along the lines: "These environments can we managed/monitored remotely hence we can make sure they are compliant with our policies."